Formula/rust.rb

```diff
@@ -88,7 +88,10 @@ def install
 
     resource("cargo").stage do
       ENV["RUSTC"] = bin/"rustc"
-      args = %W[--root #{prefix} --path . --features curl-sys/force-system-lib-on-osx]
+      args = %W[--root #{prefix} --path .]
+      on_macos do
+        args << "--features curl-sys/force-system-lib-on-osx"
```

Comment:
```suggestion
        args += %w[--features curl-sys/force-system-lib-on-osx]
```
This needs to be passed as two arguments, not one.